wagtail/tests/test_streamfield.py

```diff
@@ -237,6 +237,24 @@ def test_can_append(self):
         self.assertEqual(fetched_body[1].block_type, "text")
         self.assertEqual(fetched_body[1].value, "bar")
 
+    def test_can_append_on_queried_instance(self):
+        # The test is analog to test_can_append(), but instead of working with the
+        # in-memory version from JSONStreamModel.objects.create(), we query a fresh
+        # instance from the db.
+        # The difference is that the append below adds its data to child blocks that
+        # have not yet been lazy loaded, which caused a crash in previous versions.
```

Comment:
How about

```suggestion
        # It tests adding data to child blocks that
        # have not yet been lazy loaded. This would previously crash.
```
?

Reply:
thx, that's better :+1: